File: test/data/energyBalance/crop2ml/Algo/cs/PtSoil.cs
double AlphaE;
if (tau < tauAlpha)
{
	AlphaE = 1 ;
}
else
{
AlphaE = Alpha - ((Alpha - 1) * (1 - tau) / (1 - tauAlpha));
}
energyLimitedEvaporation= (evapoTranspirationPriestlyTaylor / Alpha) * AlphaE * tau

File: test/data/energyBalance/crop2ml/Algo/f90/PtSoil.f90
REAL AlphaE
IF (tau .LE. tauAlpha) THEN
	AlphaE = 1
ELSE
	AlphaE = Alpha - ((Alpha - 1) * (1 - tau) / (1 - tauAlpha))
ENDIF
energyLimitedEvaporation = (evapoTranspirationPriestlyTaylor / Alpha) * AlphaE * tau     
     